README.md

```diff
@@ -88,8 +88,11 @@ These containers are set to persist between sessions via the `persistent` parame
 make bufinstall && make buf 
```

Comment:
Why do we need to run `make buf` at this step.

The order I would have expected would be:
#Install brew / apt packages. Only needs to be done once.
make setup

#install buf. Only needs to be done once 
make bufinstall

#copy lua /config files into /usr/local. Only needs to be done once. Doesn't change during compilation:
make install-lua-files

#needs to be run after `make clean`
make buf

#Install the cartographer-module binary
make build

#Install the cartographer-module binaries 
make install

#Run the binary
cartographer-module